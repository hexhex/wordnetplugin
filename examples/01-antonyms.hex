%% this returns all antonyms of a word
%% POS: verbs, adjectives, adverbs

%% the search number for antonyms
%% is 1.

word("loud").
word("sleep").
word("honestly").
antonyms(P,W,A) :- &wordnet[1,W](P,_,A,_), word(W).

antonyms(2,"sleep","wake"), 
antonyms(4,"honestly","deceitfully"), 
antonyms(4,"honestly","dishonestly")?
