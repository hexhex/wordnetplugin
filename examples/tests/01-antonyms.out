{antonyms(2,"sleep","wake"), antonyms(3,"loud","soft"), antonyms(4,"honestly","deceitfully"), antonyms(4,"honestly","venally"), antonyms(4,"honestly","dishonestly"), word("honestly"), word("sleep"), word("loud")}
